bigtable/internal/testproxy/proxy.go

```diff
@@ -507,15 +529,15 @@ type goTestProxyServer struct {
 
 // client retrieves a testClient from the clientIDs map. You must lock clientsLock before calling
 // this method.
```

Comment:
Maybe move this comment into an error message?

Reply:
> You must lock clientsLock before calling this method.

Reply:
Ack. That makes sense, but is out-of-scope for this review. I'll consider your suggestion for future improvements.